pmlc/dialect/tile/transforms/padding.cc

```diff
@@ -0,0 +1,213 @@
+// Copyright 2019, Intel Corporation
+
+#include "mlir/IR/AffineExprVisitor.h"
+#include "mlir/IR/AffineMap.h"
+#include "mlir/Pass/Pass.h"
+
+#include "pmlc/dialect/eltwise/ir/ops.h"
+#include "pmlc/dialect/tile/ir/ops.h"
+#include "pmlc/dialect/tile/transforms/passes.h"
+
+namespace pmlc::dialect::tile {
+
+using mlir::AffineMap;
+
+struct BoundRange {
```

Comment:
Should it be a part of alias analysis?

Reply:
Yes I do think this will eventually move somewhere more general, but I wanted to wait until we needed a second use.  I will be building analysis structures for Affine, but for Tile I still don't have a super good idea of which parts we will use more than once, so I'd like to wait for more use case.